content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -18,9 +18,8 @@ You can use _topology spread constraints_ to control how {{< glossary_tooltip te
 
 ### Enable Feature Gate
 
-Ensure the `EvenPodsSpread` feature gate is enabled (it is disabled by default
-in 1.16). See [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/)
-for an explanation of enabling feature gates. The `EvenPodsSpread` feature gate must be enabled for the
+The `EvenPodsSpread` [feature gate] (/docs/reference/command-line-tools-reference/feature-gates/)
```

Comment:
For beta features, the gate is default to enabled, right?

Reply:
Yes, it's already enabled in source code of k/k, in 1.18.